benchmarks/src/priority_queue_push_range.cpp

```diff
@@ -59,10 +59,8 @@ void BM_push_range(benchmark::State& state) {
 
 template <size_t L>
 void putln(const benchmark::State&) {
-    static bool b = [] {
-        puts("");
-        return true;
-    }();
+    static once_flag f;
+    call_once(f, [] { puts(""); });

```

Comment:
Unused variable. Replaced to `call_once` as it is indeed unused and confusing

Reply:
It's desired to avoid magic statics in product code. But I wonder whether they should also be avoided in benchmarks. Side-effects-only static variable seems to be a common technique.

Reply:
We would need to include `<mutex>` for `call_once`. But actually I have no idea what this is intended to accomplish, as no other benchmark bothers with this. I didn't question it in #4025. I'll try to remove it.